src/scss/main.scss

```diff
@@ -1,3 +1,11 @@
 // we can keep main styles file clean by importing index files from other directories
 @import "./base";
 @import "./components";
+
+body{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  background-color: #333;
+}
```

Comment:
You shouldn't keep styles in this file, make a separate file for that and read the comments.

Reply:
This one should be fixed ☝️